how-to-capture-fiddler-trace.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How to capture Fiddler trace
+description: Step by step tutorial for capturing HTTP/HTTPS traffic with Fiddler
+type: how-to
+page_title: How to capture Fiddler trace
+slug: capture-fiddler-trace
+position: 99
+tags: fiddler, how-to
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+
+> **Service:** 3rd party tool
+> 
+> **Other stuff:** Other stuff here
+
+## Description
+This KB provides general steps for capturing HTTP/HTTPS traffic with Fiddler and saving it as a log file on the local machine. Fiddler traces are often requested by Telerik Platform support engineers, as they help tremendously in identifying issues with the Telerik Platform services on the client side. 
+
+It is important to know that Fiddler logs should be shared only in private support communication channels (e.g. support tickets) as they can consist of sensitive information. The reason for this is the fact that Fiddler captures all HTTP and HTTPS traffic when started.
+
+## Solution
+1. Download Fiddler from [here](https://www.telerik.com/download/fiddler) and install it.
+2. Run Fiddler and go to **Tools** -> **Fiddler Options**.
+3. On the **HTTPS** page, verify that **Capture HTTPS Connects** is enabled.
+4. Verify that **Decrypt HTTPS traffic** is enabled with the **_...from all processes_** option and minimize Fiddler to tray.
+5. Replicate the issue in Telerik Platform.
```

Comment:
Probably go with "Replicate the reported issue." because the issue might not be in the Telerik Platform.